administrator/components/com_joomlaupdate/controllers/update.php

```diff
@@ -41,55 +44,78 @@ public function download()
 			// Informational log only
 		}
 
+		$this->input->set('layout', 'download');
+		$this->display();
+	}
+
+	/**
+	 * Step through the download of the update package
+	 *
+	 * @return  void
+
```

Comment:
```suggestion
	 * @return  void
	 * 
```
Again CS fix, my mistake in the previous fix.